### DISCUSSION TITLE
在按照教程部署完毕后，无法运用最新版的cuda部署MinerU

### DISCUSSION THREAD

**Discussion by MKmark946076** [magicpdf/Magic-PDF#514] [Q&A]
![torch](https://github.com/user-attachments/assets/47ba5937-1093-4aa6-8630-12b0afa0b193)
您好，
在按照教程部署完GPU运行环境后，报错，提示“Torch not compiled with CUDA enabled”

**Comment by Zzkuangg** [comment]
我的cuda版本高于11.8,这种情况是不能正确运行对吗

**Comment by myhloli** [comment]
> ![torch](https://private-user-images.githubusercontent.com/129722407/363419935-47ba5937-1093-4aa6-8630-12b0afa0b193.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU5MDA4MjksIm5iZiI6MTcyNTkwMDUyOSwicGF0aCI6Ii8xMjk3MjI0MDcvMzYzNDE5OTM1LTQ3YmE1OTM3LTEwOTMtNGFhNi04NjMwLTEyYjBhZmEwYjE5My5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTA5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkwOVQxNjQ4NDlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00NThiNGQxYTZmNTdkZDgwNmRlMGY5OTM3M2EyMmQwNDUxOTViYjQyYTNlZTdjYWM4ZDkxMmQ1NmYxYzUwMDRiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.E7sBOw0rJV4hcOI1pPubpsDI0we0beSNqiIPA_gRuio) 您好， 在按照教程部署完GPU运行环境后，报错，提示“Torch not compiled with CUDA enabled”

pip install --force-reinstall torch==2.3.1 torchvision==0.18.1 --index-url https://download.pytorch.org/whl/cu118
